### DISCUSSION TITLE
How do I override the generated social card (og:image)?

### DISCUSSION THREAD

**Discussion by ssbarnea** [squidfunk/mkdocs-material#5162] [Q&A] [answered]
As the generated social card does not seem to be rich enough for some projects, I am wondering how can I configure a default image to use for it??

A classic example would be configuring the the [github social preview image](https://docs.github.com/en/repositories/managing-your-repositorys-settings-and-features/customizing-your-repository/customizing-your-repositorys-social-media-preview) but I did not find a way to do it.

I observed that the theme does add something like below where the page name is "README.md", still even if I have my own social-preview image, I do not know how to force it to be used for a specific page or for the entire website.

```html
<meta property="og:image" content="http://127.0.0.1:8000/assets/images/social/README.png">
```

**Comment by squidfunk** [comment, marked as answer]
See https://github.com/squidfunk/mkdocs-material/issues/4402#issuecomment-1457812285 for my original answer.
